extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -99,7 +89,8 @@ export class WindowPortEmulator {
    * @param {JsonObject} data
    */
   postMessage(data) {
-    this.target_./*OK*/ postMessage(data, this.origin_);
+    const targetOrigin = this.origin_ === 'null' ? '*' : this.origin_;
```

Comment:
Please clarify this change. Why is it needed now?

Reply:
Added comment to explain opaque origin.